mmv1/templates/terraform/examples/clouddeploy_custom_target_type_gcb_repo_skaffold_modules.tf.erb

```diff
@@ -0,0 +1,17 @@
+resource "google_clouddeploy_custom_target_type" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
You'll need to register this example in the `CustomTargetType.yaml` file. More details at https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-a-create-test

Reply:
Sorry I was probably not clear previously. You'll need to add something like https://github.com/katiexzhang/magic-modules/blob/main/mmv1/products/clouddeploy/CustomTargetType.yaml#L57-L62 in in the `CustomTargetType.yaml` file so that this example will be generated in both tests and web docs. 

Plus, you can follow https://googlecloudplatform.github.io/magic-modules/get-started/generate-providers/ to generate providers and https://googlecloudplatform.github.io/magic-modules/develop/test/run-tests/#run-tests to run tests locally which may help you view the test result faster.

Please let me know if you have any questions.

Reply:
Sorry I was probably not clear previously. You'll need to add something like https://github.com/katiexzhang/magic-modules/blob/main/mmv1/products/clouddeploy/CustomTargetType.yaml#L57-L62 in in the `CustomTargetType.yaml` file so that this example will be generated in both tests and web docs.